src/Core/src/Graphics/Rectangle.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Graphics;
+
+namespace Microsoft.Maui.Graphics
+{
+	public class Rectangle : Shape
+	{
+		public Rectangle()
+		{
+			Stretch = Stretch.Fill;
+		}
+
+		public Rectangle(CornerRadius cornerRadius) : this()
+		{
+			CornerRadius = cornerRadius;
+		}
+
+		public CornerRadius CornerRadius { get; set; }
```

Comment:
I have replaced `RadiusX` and `RadiusY` with `CornerRadius`. From all the feedabck received in Xamarin.Forms it seems like what most devs really expect.